src/main/java/Data/User.java

```diff
@@ -0,0 +1,63 @@
+package Data;
+
+public class User {
+
+    String username;
```

Comment:
it should be private

Reply:
Moreover User class is used in User Manager so why should it be private? I don't understand.